Apollo/ApolloExtra.lua

```diff
@@ -3558,20 +3558,44 @@ OlympusTraitData.MasterLobDionysusTrait =
 					PlayOnce = true,
 					PreEventFunctionName = "BoonInteractPresentation", PreEventFunctionArgs = { PickupWait = 1.0, },
 					RequiredTextLines = { "PersephoneMeeting02", "ApolloBackstory02" },
+					RequiredFalseTextLines = { "PersephoneReturnsHome01", "ApolloBackstory03b" },
 					{ Cue = "/VO/Apollo_0092",
 						StartSound = "/Leftovers/World Sounds/MapZoomInShort", UseEventEndSound = true,
 						Text = "I try not to eavesdrop, but I heard that Hades once had a Queen. I assumed it was Nyx, but I had a vision lately and...  Oh, I won't give away to much of the fun. You'll see what I mean." },
 				},
+				ApolloBackstory03b =
```

Comment:
Those 2 new Voice lines for Apollo are not added to VO.fsb for now since we are waiting for Edit. They'll just stay silent for the time being.